### ISSUE TITLE
[Bug]: project doesn't load, csv-file does load

### ISSUE THREAD

**Issue by SabrinaKen** [jasp-stats/jasp-issues#2969]
### JASP Version

0.19.1

### JASP Module

Other

### What OS are you seeing the problem on?

Windows 11

### Bug Description

I cannot open my jasp-project that I created during the last months. When I try to open a new csv-file, it works but not my project. JASP is not responding while I try to open my project. When I open the task manager, there is no JASP Engine starting. If I open another CSV-file, the enging is starting. My brother is trying to open it on the same remote server I am using but with his laptop and everything is working fine. 
![Project does not load](https://github.com/user-attachments/assets/eb510365-414e-4ec2-99a6-08f0a4920ea2)
![Task Manager](https://github.com/user-attachments/assets/5654448d-c4ae-46c3-850d-70c42962267c)

### Expected Behaviour

The project should open.

### Steps to Reproduce

1. Open JASP
2. Open File
3. It is loading
4. It is not opening, but the programm "is not responding" anymore

### Log (if any)

sensitive information on file

### Final Checklist

- [ ] I have included a screenshot showcasing the issue, if possible.
- [ ] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [ ] I have accurately described the bug, and steps to reproduce it.

**Comment by boutinb** [comment]
Hi @SabrinaKen Can you send your JASP file? Or at least the log files: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/user-guide/logging-howto.md ?

**Comment by SabrinaKen** [comment]
[JASP 2024-10-16 08_53_33 Desktop.log](https://github.com/user-attachments/files/17392677/JASP.2024-10-16.08_53_33.Desktop.log)
[JASP 2024-10-16 08_53_33 Engine 0.log](https://github.com/user-attachments/files/17392678/JASP.2024-10-16.08_53_33.Engine.0.log)
[JASP 2024-10-16 08_59_51 Desktop.log](https://github.com/user-attachments/files/17392680/JASP.2024-10-16.08_59_51.Desktop.log)
[JASP 2024-10-16 08_59_51 Engine 0.log](https://github.com/user-attachments/files/17392681/JASP.2024-10-16.08_59_51.Engine.0.log)

Is this the right thing, you asked for?

**Comment by hugoquene** [comment]
Some students in my course on Methods and Statistics report the same issue in using JASP 0.19.1... A few students cannot open an already existing JASP project file which contains data and analyses.

**Comment by boutinb** [comment]
The logs do not say so much. Could I have a jasp file that you cannot load? You have to zip it, and drag-and-drop it here.
If you prefer you can also send the JASP file to our email address [info@jasp-stats.org](mailto:info@jasp-stats.org)

**Comment by boutinb** [comment]
Hi @SabrinaKen I can open your JASP file without problem... There are nearly 30 analyses in one file, maybe it's a bit too much for your PC. So I'm going to send you a JASP file with less analyses (15).

**Comment by boutinb** [comment]
@hugoquene For @SabrinaKen the problem was that apparently her PC had not enough memory to load all the analyses (30) from her JASP file. If I can get also your JASP file, I can see it is the problem for you also.

**Comment by tomtomme** [comment]
Maybe we should add a warning when one adds more than 20 Analysis within jasp

**Comment by EJWagenmakers** [comment]
Yes. I assume the problem is a function of file size? We could also issue a warning whenever the file size exceeds a certain number....

**Comment by tomtomme** [comment]
@SabrinaKen 
could you change the title of this issue to [Feature Request] Show warning PopUp when exceeding 20 analysis
